debian/control

```diff
@@ -23,7 +23,6 @@ Build-Depends:
  libsoup2.4-dev,
  libsystemd-dev,
  ostree,
- xsltproc,
```

Comment:
I was doing separate commits for changes in the debian directory for the sake of "debian: " prefix in the commit message. :)